### PR TITLE
[Impeller] match sigma scaling to Skia scaling.

### PR DESCRIPTION
**Pull request by jonahwilliams**
The Impeller blur is scaling down about twice as fast as the Skia Blur. Instead adopt their approximate constant values, which results in less shaking as the blur gets bigger.

FYI @gaaclarke

### CODE REVIEW

**Comment by jonahwilliams** [comment]
From some napkin math, on the wonderous app the current scale down was going down to a sigma of ~2, where the Skia scaling curve never scales below 4.

File: impeller/entity/contents/filters/gaussian_blur_filter_contents.cc [hunk 1]
```diff
@@ -144,21 +144,23 @@ std::shared_ptr<Texture> MakeBlurSubpass(
...
 }  // namespace
 
 GaussianBlurFilterContents::GaussianBlurFilterContents(Scalar sigma)
     : sigma_(sigma) {}
 
+/// Calculate how much to scale down the texture depending on the blur radius.
+///
+/// This value was extracted from Skia, see:
+///  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/ganesh/GrBlurUtils.cpp#L2561-L2576
+///  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/BlurUtils.h#L57
```
Comment (gaaclarke) [inline impeller/entity/contents/filters/gaussian_blur_filter_contents.cc:156]: No need to add another docstring.
```suggestion
// This value was extracted from Skia, see:
//  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/ganesh/GrBlurUtils.cpp#L2561-L2576
//  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/BlurUtils.h#L57
```

File: impeller/entity/contents/filters/gaussian_blur_filter_contents.h [hunk 2]
```diff
@@ -42,6 +42,11 @@ class GaussianBlurFilterContents final : public FilterContents {
                            const Entity& entity,
                            const ISize& pass_size);
 
+  /// Calculate the scale factor for the given sigma value.
```
Comment (gaaclarke) [inline impeller/entity/contents/filters/gaussian_blur_filter_contents.h:45]:
```suggestion
  /// Calculate the scale factor for the downsample pass given a sigma value.
```

**Review by gaaclarke — APPROVED** [review]
LGTM, thanks.  I wonder if we are still happy with the performance 🤞

**Comment by jonahwilliams** [comment]
I think that we could further adjust the scale down - we just need to do so in a way that keeps the sigma from getting smaller than 4-ish, because that is when things start getting jittery.

**Pull request merged**